docs/how-to/enable_hugepages_for_upf.md

```diff
@@ -0,0 +1,38 @@
+# Enable HugePages in the UPF
+
+## Requirements
+
+- A Kubernetes Host 
+  - CPU that supports AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+  - CPU has the flag "pdpe1gb" which allows to set 1Gi HugePages
+  - HugePages is enabled and has enough 2Gi free HugePages
+- A Kubernetes cluster with the Multus addon enabled
+- Juju = 3.1/stable
```

Comment:
```suggestion
- Juju >= 3.1/stable
```

Reply:
This documentation needs to be in sync with core network deployment. Juju = 3.1/stable is used because the scdore-tests  uses it as a direct requirement.  We can not be sure if the  >= 3.1/stable is going to break something without testing it.
https://github.com/canonical/sdcore-tests/blob/main/.github/workflows/integration-tests.yaml#L67

Reply:
The intent is not to update our docs every time a new minor version of Juju is released. If our charms work with Juju 3.1 and not with Juju 3.2 it will be because of a bug in Juju.

Reply:
OK, it is changed.